src/Tasks/Hash.cs

```diff
@@ -37,61 +35,120 @@ public class Hash : TaskExtension
         [Output]
         public string HashResult { get; set; }
 
+        private static readonly char ItemSeparatorCharacter = '\u2028';
+
+        private static readonly Encoding encoding = Encoding.UTF8;
+
+        private static readonly byte[] ItemSeparatorCharacterBytes = encoding.GetBytes(new char[] { ItemSeparatorCharacter });
+
```

Comment:
Why static vs. constant? Aren't these all used the same way?

Reply:
I wanted to allocate and compute this byte array only once, so I opted for `static readonly` instead of `const`. Two other static variables are the ones the byte array depends on. There is no reason though for two other variables not being static too, as far as I see.